### ISSUE TITLE
Enhancement: CIFS support

### ISSUE THREAD

**Issue by deed02392** [Hani-K/H-Vitamin#1]
Creating issue to capture a request for CIFS module to be included with an H-Vitamin kernel build to enable wifi SMB sharing of files to and from phones.

**Comment by deed02392** [comment]
@Hani-K I noticed the source is under `fs/cifs` already - I haven't checked if it's compiled yet, though

**Comment by Hani-K** [comment]
@deed02392 No, it's not, will add a few modifications and enhancements later, including the CIFS support.

**Comment by deed02392** [comment]
I have a 910U and Sue max currently due to this kernel not supporting my modem. Can test with you if you like.
